groupmne/__init__.py

```diff
@@ -1,12 +1,11 @@
 """Multi-subject source localization with MNE."""
 
 from . import group_model, inverse, utils
-from .group_model import (compute_gains, compute_inv_data, get_src_reference,
-                          compute_fwd)
-from .inverse import compute_group_inverse
+from .group_model import compute_gains, get_src_reference, compute_fwd
+from .inverse import InverseOperator
 from ._version import __version__
 
 
-__all__ = ["group_model", "inverse", "compute_gains", "compute_inv_data",
-           "get_src_reference", "compute_group_inverse", "compute_fwd",
+__all__ = ["group_model", "inverse", "compute_gains",
+           "get_src_reference", "InverseOperator", "compute_fwd",
```

Comment:
only put in the public API what is used in the documentation / examples. The rest is just for you and your internal cooking.

Reply:
Yes